iothub_client/src/iothub_client_authorization.c

```diff
@@ -314,7 +314,7 @@ char* IoTHubClient_Auth_Get_SasToken(IOTHUB_AUTHORIZATION_HANDLE handle, const c
             }
             else 
             {
-                size_t expiry_time = sec_since_epoch+expire_time;
+                size_t expiry_time = sec_since_epoch+expiry_time_relative_seconds;
```

Comment:
>+expiry_time_relative_seconds; [](start = 52, length = 30)

Not your fault but I find it very irritating that we are basically doing this computation twice in this function.